modules/widgets/src/reset-view-widget.tsx

```diff
@@ -61,18 +64,10 @@ export class ResetViewWidget extends Widget<ResetViewWidgetProps> {
 
   handleClick() {
     const initialViewState = this.props.initialViewState || this.deck?.props.initialViewState;
```

Comment:
We really ought to combine what this widget does and what compass widget does so that they can be used in either mode (initialViewState/viewState)

A user managing viewState would be able to use this widget too if we made `homeViewState` a widget prop, and stored `viewports` in the widget like the others.

In compass, we don't need to expose a prop but just check if `initialViewState` is in use.